libs/services/designer-client-services/src/lib/standard/run.ts

```diff
@@ -226,13 +228,22 @@ export class StandardRunService implements IRunService {
       return Promise.resolve({ inputs: this.parseActionLink(inputs, true), outputs: this.parseActionLink(outputs, false) });
     }
 
-    if (outputsLink && outputsLink.uri) {
-      outputs = await this.getContent(outputsLink);
-    }
-    if (inputsLink && inputsLink.uri) {
-      inputs = await this.getContent(inputsLink);
+    try {
+      if (outputsLink && outputsLink.uri) {
+        outputs = await this.getContent(outputsLink);
+      }
+      if (inputsLink && inputsLink.uri) {
+        inputs = await this.getContent(inputsLink);
+      }
+    } catch (e: any) {
+      if (e.message.includes('Failed to fetch') && !isNullOrUndefined(updateCors)) {
```

Comment:
If anyone knows a better way to catch a specific denied by CORS api call rather than just by the api message would be much appreciate it

Reply:
Not really a CORS failure essentially means the call never actually happens at all